app/assets/javascripts/reports.js.coffee

```diff
@@ -0,0 +1,31 @@
+ready = ->
+  $ ->
+    $(':submit').hide()
+    $('#report-selector').change ->
+      $('#genre-selector').remove()
+      if $('#report-selector').val() == "book-popularity"
+        $.ajax
+          type: "get",
+          url: '/genres.json'
+          success: (data, status, response) ->
+            selector = "<fieldset  id='genre-selector'>
+                          <label>Select a Category</label>
+                          <select name='genre' required='required'>
+                            <option></option>
+                            <option value=''>All Categories</option>"
```

Comment:
I think we need to give this a value other than '', in Firefox, I'm still getting prompted to select an option when I choose 'all categories'